packages/scxa-marker-gene-heatmap/src/MarkerGeneHeatmap.js

```diff
@@ -168,7 +169,13 @@ const MarkerGeneHeatmap = (props) => {
       minorGridLineWidth: 0,
       plotLines: plotLines,
       showEmpty: false,
-      visible: data.length !== 0
+      visible: data.length !== 0,
+      labels: {
+        useHTML: true,
+        formatter: function() {
+          return YAxisFormatter(species, this.value)
```

Comment:
I think that `YAxisFormatter` adds a huge amount of complexity that’s totally uncalled for. How about this?
```js
formatter: function() {
  return `<a href="https://www.ebi.ac.uk/gxa/sc/search?q=${this.value}&species=${species}"` +
    `style="border: nonel; color: #148ff3">${this.value}</a>`
}
```